packages/addons/src/paths.ts

```diff
@@ -14,30 +14,107 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import type { BpmnElementsRegistry, ShapeBpmnSemantic } from 'bpmn-visualization';
+import type { EdgeBpmnSemantic, ElementsRegistry, ShapeBpmnSemantic } from 'bpmn-visualization';
+
+// bpmn-visualization does not filter duplicates when passing an ids several times
+const filterDuplicates = (ids: string[]): string[] => [...new Set(ids)];
```

Comment:
**thought:** IMHO this is a bug in `bpmn-visualization` that we should fix. It applies to all methods of ElementsRegistry taking ids or kinds.
What do you think @csouchet?
If so, I will create an issue.

Reply:
Indeed, the APIs of `bpmn-visualization` should check une unicity of the parameter.

Reply:
Fine, prior to merge this PR, I am going to create a dedicated issue in `bpmn-visualization` and another one in `bv-experimental-add-ons` to update the implementation when the fix is available in a new release of `bpmn-visualization`.

Reply:
Bug created: https://github.com/process-analytics/bpmn-visualization-js/issues/2921
Refactor task created: #150